vault/helm/vault/values.yaml.tpl

```diff
@@ -43,7 +43,11 @@ vault:
         cert-manager.io/cluster-issuer: letsencrypt-prod
         nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
         nginx.ingress.kubernetes.io/use-regex: "true"
+      {{- if not .Values.exposePrivate }}
       ingressClassName: nginx
```

Comment:
A lot of these values should actually be moved down to values.yaml since the current setup prevents reconfiguration.  Really we should have nginx as the default there then overwrite with:

{{ if .Values.exposePrivate }}
ingressClassName: internal-nginx
{{ end }}

in `values.yaml.tpl`

Reply:
Sure. I moved the values to values.yaml.tpl. Ihad to leave also the else statement, as otherwise the className gets undefined.